app/gui/view/graph-editor/src/component/visualization/container/visualization_chooser.rs

```diff
@@ -4,7 +4,8 @@
 //!
 //! TODO: If similar things are needed elsewhere, refactor this to a
 //! Chooser<T:Eq+Display> (or similar) which would represent a `DropDownMenu` for specific owned
-//! values.
+//! values. It should also be refactored to use `drop_down_menu` from `ensogl_components` instead
+//! of the old list view.
```

Comment:
Why are we not using the new drop-down in the first place here?

Reply:
The "new drop-down" only implements the part that opens and closes, while this component has the "clickable selected item". Ideally, they should be refactored to be merged, but for this use cases, the "old one" is more appropriate (and the additional refactoring not required at this time).